### ISSUE TITLE
new feature: explicitly set range and default value

### ISSUE THREAD

**Issue by ginatrapani** [postlight/account#8]
Right now you can't set a slider to default to say, 3, but make the range 1-10. If you set it to 3, the range only goes to 6. If you set it to 1-10, the default is 5.

**Comment by ftrain** [comment]
So you could have:

10 (default behavior is to set range min 0 max 20 default 10)
1-10 (set range min 1 max 10)
1-5-10 (set range min 1 max 10 default 5)

it makes parsing negatives (both actually and visually) harder so maybe

1
-1...10
-10...-5...10

or 
-100
-100 - -100
-100 - -50 - 100

or

-100
-100 to -10
-100 to -50 to 100

idk, thinking

**Comment by ginatrapani** [comment]
yeah I dislike `-` as the range notation, I wish it was `1..10` instead of `1-10`

Extending on that, I'd love to indicate the default like this:

`1..3..10` where 3 is the default.

**Comment by ginatrapani** [comment]
or 3 dots, either way
